.tidyoperators

```diff
@@ -0,0 +1,7 @@
+Text.Parsing.Indent.(<*/>) 11
+Text.Parsing.Indent.(<+/>) 9
+Text.Parsing.Indent.(<-/>) 10
+Text.Parsing.Indent.(<?/>) 12
+Text.Parsing.Parser.Combinators.(<?>) 4
+Text.Parsing.Parser.Combinators.(<??>) 3
+Text.Parsing.Parser.Combinators.(<~?>) 4
```

Comment:
You need to add all GH sources.

Reply:
ie, the operator tables don't inherit from the default set.

Reply:
Huh... I ran `purs-tidy generate-operators "$(spago sources)" > .tidyoperators` to get the above output.
After reading the above comment, I then tried what your readme says:
```
spago sources | xargs purs-tidy generate-operators > .tidyoperators
```
I still get the same thing.

Reply:
Maybe xargs behave differently on your system? I just tried it and I get the full prelude in there.

Reply:
This is what I got https://gist.github.com/natefaubion/5fadb674d643666272aa07959559880e